submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,105 @@
+
+If you're intrigued and want to hear more about the specifics of mall design, then read on as Sir Fendi takes a deep dive into the various approaches and techniques seen around the community.
+
+
+
+### Mall Design Factors <author>Sir Fendi</author>
+
+While all malls share the common purpose of minimizing handcrafting, they come in various shapes and sizes depending on what you want and which technologies you have unlocked. Whether you improvise together a mall or painstakingly prepare a blueprint for it, there are a number of design aspects for which you have multiple options. I would like to shine a light on these various design factors:
+
+## **Is the mall integrated or dedicated?**
+
+One way to build a mall is to integrate it into your science production. A basic example of this is adding an output chest to your transport belt and inserter assembling machines that normally produce these items for logistics science packs. The advantage of this approach is that your mall has an easy supply of intermediate items that would normally be produced anyway for your science packs. Hence, you are saved from having to build additional infrastructure. Using this approach you can build an all-in-one compact early game base, which makes it easier to rush through the first few science packs. 
+
+[Integrated mall example: The "Jump Start Base" by Nilaus]
+
+[Integrated mall example: My TINS starter base]
+
+A consequence of the integrated design is your mall machines interrupting your science production as they resupply their outputs, which may be something you prefer already if you want research and factory building to happen in parallel. These interruptions are most severe upon first building the mall and are much less later on, because many of the mall machines tend to be idle more often than not. A disadvantage of improvising an integrated design is that you can run low on space and spaghettify pretty quickly.
+
+The alternative approach is building a dedicated area for your mall. Without relying on your science production, a dedicated mall requires its own resource supply logistics, which can be belts branching off a main bus or dedicated resupply trains/bots. Alternatively the area can have its own intermediate item production area, perhaps even including its own furnaces such that it only needs ores. A dedicated mall area is easier to lay out and expand compared to an integrated area, especially when improvising. This approach also makes it easier to control how much of your resources you want to divert to science production versus mall production.
+
+[Dedicated mall example from community]
```

Comment:
KoS was also a big provider of mall BP, and still (widely?) used based on what I can see on different MP games

Reply:
(visual example from KoS)